setup.py

```diff
@@ -35,8 +34,6 @@
 LONG_DESCRIPTION = re.sub(r'(?s)\[(.*?)\]\((http.*?)\)', r'\1', LONG_DESCRIPTION_MD)
 
 INSTALL_REQUIRES = ["setuptools", "idna", "requests>=2.1.0", "requests-file>=1.4"]
-if (2, 7) > sys.version_info:
-    INSTALL_REQUIRES.append("argparse>=1.2.1")
```

Comment:
@hugovk so in light of the other comment, would you mind, instead of deleting line 39 here, actually raise an error like `RuntimeError`? to hard-block people from installing if they are on 2.6.

Reply:
Done! Or I've put the warning right near the top of the file to make sure it's executed right away, and before any potential syntax errors in later code that may only occur on 2.6.